{"name": "Невозможность изменения данных неавторизованного пользователя, email", "status": "passed", "description": "При попытке изменения возвращает HTTP 401 и сообщение об ошибке", "steps": [{"name": "Отправка POST на auth/login", "status": "passed", "parameters": [{"name": "payload", "value": "{'email': 'aknox@example.com', 'password': '5792273348', 'name': 'Robert Cooke'}"}], "start": 1718266320830, "stop": 1718266321093}, {"name": "Отправка PATCH на auth/user", "status": "passed", "parameters": [{"name": "payload", "value": "{'email': 'abracadabra', 'password': '5792273348', 'name': 'Robert Cooke'}"}, {"name": "access_token", "value": "''"}], "start": 1718266321093, "stop": 1718266321322}], "parameters": [{"name": "field", "value": "'email'"}], "start": 1718266320830, "stop": 1718266321323, "uuid": "e0067f79-bfac-42b8-a6d9-6e523bbf4788", "historyId": "7b7719bfa965413a1d89dabc1cb3e74c", "testCaseId": "ed8b5b9382a269d205c24e64879389b0", "fullName": "tests.test_auth_user.TestChangeUserData#test_change_unauthorized_user_data_failure", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_auth_user"}, {"name": "subSuite", "value": "TestChangeUserData"}, {"name": "host", "value": "XPMEPODW0067"}, {"name": "thread", "value": "28304-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_auth_user"}]}